### DISCUSSION TITLE
Movie scrape??

### DISCUSSION THREAD

**Discussion by Authority726** [stashapp/stash#4913] [Q&A] [answered]
Hi,

I dont really understand how movie section works. I see a lot of scrapers and stuff but i can no where see any action/button for me to scrape using movie. How can i for example scrape an URL to create a new movie ? (Not scene).

Regards

**Comment by WithoutPants** [comment, marked as answer]
Movies can only be scraped via URL. The process is initiated by pasting in a URL in the URL field, then clicking on the scrape button on the right side of the URL field. You can see the supported URLs under the `Scrapers -> Movie scrapers` section of the `Metadata Providers` setting panel.

**Comment by Authority726** [comment]
Thanks! Is it possible to create placeholder movies with just the URL by script?

**Reply by stg-annon** [reply]
Yes movies are essentially just a container for scenes at the moment, future updates will change this

you will first need to install the scraper for the movie, likely from https://github.com/stashapp/CommunityScripts

the manual process would be go to the Movies tab select New paste the URL into the URL field then click the button to the right of the URL field this will scrape the URL and populate the metadata for the Movie then click Save

to script this you would chain a `scrapeMovieURL` query with a `movieCreate` mutation see `localhost:9999/playground` for details

If this goes over you head you might want to look at [Serechops/Movie-Fy Bulk Movie URL Scrape](https://github.com/Serechops/Serechops-Stash/tree/main/plugins/Movie-Fy/Movie-Fy%20Bulk%20Movie%20URL%20Scrape)
